client-v2/README.md

```diff
@@ -19,8 +37,46 @@ To start developing just run:
 The v2 application will be running here:
 [https://fronts.local.dev-gutools.co.uk/v2](https://fronts.local.dev-gutools.co.uk/v2)
 
+View [client-dev.sh](scripts/client-dev.sh).
+
+### Technologies
+
+V2 is a ReactRedux Javascript application hooking into the existing Fronts API and CAPI.
+
+You'll need to understand the concepts of Thunks and Selectors.
+
+| Uses        | For         | Config |
+| ------------|------------- |--- |
+| [React](https://jestjs.io/docs/en/getting-started.html)      | Components ||
+| [Redux](https://github.com/airbnb/enzyme)      | State management ||
+| [Redux Thunk](https://github.com/reduxjs/redux-thunk)      | Redux Thunk middleware allows you to write action creators that return a function instead of an action.  ||
+| [Reselect](https://github.com/reduxjs/reselect)      | Selectors can compute derived data, allowing Redux to store the minimal possible state.  ||
+| [Redux Forms](https://redux-form.com/7.4.2/)      | Form state management for Redux apps ||
+| [React Router](https://reacttraining.com/react-router/)      | Routing for React apps ||
+| [Styled Components](https://www.styled-components.com/)      | Styling for components ||
+| [Typescript](https://www.typescriptlang.org/)      | JS Types | [tsconfig](tsconfig.json), [tslint](tslint.json), [modules.d.ts](modules.d.ts), [global.d.ts](global.d.ts)|
+| [Lodash](https://lodash.com/)      | JS helper utilities ||
+| [Date Fns](https://date-fns.org/)      | JS Date functions ||
+| [Raven](https://github.com/getsentry/sentry-javascript)      | Sentry error reporting ||
+| [Panda Session](https://github.com/guardian/panda-session)      | Pan Domain (cross-gutools) session management ||
+
+## Building and Compiling
+
+| Uses        | For         |Config |
+| ------------|------------- |---|
+| [Yarn](https://yarnpkg.com/en/)      | Yarning | [package.json](package.json) |
+| [Babel](https://babeljs.io/)      | Compile Typescript etc | [babel.config.json](babel.config.json) |
+| [Webpack](https://webpack.js.org/)      | Bundle your assests | [webpack.config.common.js](config/webpack.config.common.js), [webpack.config.prod.js](config/webpack.config.prod.js), [webpack.config.dev.js](config/webpack.config.dev.js) |
+
 ## Testing
 
+| Uses        | For         | Config |
+| ------------|------------- |--- |
+| [Jest](https://jestjs.io/docs/en/getting-started.html)      | Testing library | [jest.config.js](jest.config.js)|
+| [Enzyme](https://github.com/airbnb/enzyme)      | JavaScript Testing utilities for React  ||
+| [React Test Renderer](https://reactjs.org/docs/test-renderer.html)      | Render Components to JSON (e.g for Jest Snapshotting) ||
+| [Test Cafe](https://testcafe.devexpress.com/)      | Integration testing - see below ||
+
```

Comment:
As mentioned here we could throw in `react-testing-libaray` here. @vlbee is that something you can just add into this(with GitHub's new suggested change!?) maybe unless @gtrufitt is already on it?!

Reply:
```suggestion
| [Enzyme](https://github.com/airbnb/enzyme)      | JavaScript Testing utilities for React  ||
| [react-testing-library](https://github.com/kentcdodds/react-testing-library)     | JavaScript Testing utilities for React Components |
```

Reply:
I've added the suggestion further down the conversation.